src/components/common/HelpSection.vue

```diff
@@ -76,6 +76,10 @@ export default class HelpSection extends Vue {
   justify-content: center;
 }
 
+.help-header .help-header-h2 {
+  letter-spacing: 0;
```

Comment:
I'm thinking that all h2 elements should have zero letter-spacing. Please check with UI team. If true then you should remove it globally in base.scss.

Also ask about h1 letter-spacing since maybe that should be removed globally also.

(Compare with base.scss in Create UI.)

Reply:
I am confirming it with Tracy now. I'll update the base.scss file once she confirms.

Reply:
Great. I'm assuming the base h1 and h2 styles should match with Create UI.

Reply:
Tracy has confirmed that h1 and h2 should use 0 letter spacing.